server/routes/api/choropleth.py

```diff
@@ -59,7 +59,8 @@ def choropleth_values():
                                  flask.request.args.get("level"))
     if not display_level:
         return flask.jsonify({"error":
-           f"Failed to automatically resolve subgeo for {dcid}."}, 400) 
+           f"Failed to automatically resolve subgeo for {dcid}." +
```

Comment:
"subgeo" might not be an easily understood term.  I am not sure what a good term is.  ""geographic subdivision level" is a guess.